### DISCUSSION TITLE
Helping with development

### DISCUSSION THREAD

**Discussion by DaxamLhant** [phst-randomizer/ph-randomizer#443] [General]
Hello,

I've been checking in on this project every so often and I'm wondering if there is a way I can get more involved with it. I have helped a bit here and there with other randomizers mostly doing some testing and working on logic. I have some experience with python and should have plenty of time to be able to help with reverse engineering the games.

Please let me know if theres a way to get in contact to assist.

Thank you.

**Comment by Arrobadeibid** [comment]
Hi,

Same thing, although Im not a programmer, I have virtually no experience with coding. What I am is a bilingual Spanish-English translator that would be more than happy to translate the program and website/discord server/whatever (obvs for free) when you guys have it up and running! Also if betatesting is needed Im down to help.

**Comment by mike8699** [comment]
Hi there, sorry I missed this post until now.

There's not much in the way of pure python coding tasking left, and testing is still a long way off. Currently, the priority is finishing up the ROM patches needed for the randomizer (editing event scripts, ASM hacks, etc.). The [issue tracker](https://github.com/phst-randomizer/ph-randomizer/issues) is up to date, so feel free to take a look there and see if there's anything there you think you could work on.
